azure-pipelines.yml

```diff
@@ -37,21 +38,15 @@ stages:
         scannerMode: 'MSBuild'
         projectKey: '2011-fakebook-project3_profile'
         projectName: 'profile'
-        extraProperties: |
-          sonar.exclusions=**/lib/**
-          sonar.cs.opencover.reportsPaths=$(Agent.TempDirectory)/**/coverage.opencover.xml
-
+        extraProperties: 'sonar.cs.opencover.reportsPaths=$(Agent.TempDirectory)/**/coverage.opencover.xml'
+        
     - task: UseDotNet@2
       displayName: 'Use .NET 5 SDK'
       inputs:
         packageType: 'sdk'
         version: $(sdkVersion)
         installationPath: $(Agent.ToolsDirectory)/dotnet
-
-    - script: dotnet build --configuration $(buildConfiguration)
-      displayName: dotnet build
-      workingDirectory: $(targetSolution)
-      
+        
```

Comment:
you still need the dotnet build because dotnet test in theory might not compile every project